internal/donor/donordata/provided.go

```diff
@@ -45,20 +45,20 @@ type Tasks struct {
 
 // Provided contains all the data related to the LPA application
 type Provided struct {
-	PK dynamo.LpaKeyType      `hash:"-"`
-	SK dynamo.LpaOwnerKeyType `hash:"-"`
+	PK dynamo.LpaKeyType      `hash:"-" checkhash:"-"`
```

Comment:
annoying that we'll have to duplicate tags, but this was the only way I could think to allow the nested things to know whether we want the full hash or the checked hash